demisto_sdk/commands/generate_test_playbook/test_playbook_generator.py

```diff
@@ -159,13 +159,13 @@ def create_automation_task(_id, automation_name, item_type: ContentItemType, arg
     }
 
 
-def create_verify_outputs_task(id, conditions=[]):
+def create_verify_outputs_task(id_, conditions=[]):
```

Comment:
Why the arg name change?

Reply:
id-is-a-bad-variable-name-in-python

https://stackoverflow.com/questions/77552/id-is-a-bad-variable-name-in-python#:~:text=It's%20bad%20to%20name%20any,know%20the%20name%20is%20overridden.&text=id%20is%20a%20built%2Din%20function%20in%20Python.,id%20will%20override%20the%20function.